docs/src/__examples__/ButtonPrimitive/DEFAULT.tsx

```diff
@@ -14,9 +14,142 @@ export default {
       Click me
     </ButtonPrimitive>
   ),
-  info: {
-    title: "Default button primitive",
-    description:
-      "The default button primitive appears as a &lt;button&gt; with other traits inherited or as the browser default.",
-  },
+  exampleKnobs: [
+    {
+      component: "ButtonPrimitive",
+      knobs: [
+        {
+          name: "iconLeft",
+          type: "icon",
+          defaultValue: "",
+        },
+        {
+          name: "iconRight",
+          type: "icon",
+          defaultValue: "",
+        },
+        {
+          name: "icons",
+          type: "icon",
+          defaultValue: "",
+        },
+        {
+          name: "submit",
+          type: "boolean",
+          defaultValue: false,
+        },
+        {
+          name: "fullWidth",
+          type: "boolean",
+          defaultValue: false,
+        },
+        {
+          name: "title",
+          type: "text",
+          defaultValue: "",
+        },
+        {
+          name: "height",
+          type: "text",
+          defaultValue: "",
+        },
+        {
+          name: "boxShadow",
+          type: "text",
+          defaultValue: "",
+        },
+        {
+          name: "boxShadowHover",
+          type: "text",
+          defaultValue: "",
+        },
```

Comment:
I don't think it makes sense to add knobs which require writing code, they have the code editor for that. I'd only add a few knobs for ButtonPrimitive, the rest of them are a little much, not sure who would use those.